mca/identifiers.py

```diff
@@ -83,7 +103,31 @@ def get_discovery_flow_ids(self, hop, ip):
 
         return unique_flow_ids
 
-    def create_new_discovery_flow_id(self, hop, ignore=[]):
+    def create_new_discovery_flow_id(self,
+                                     hop: int,
+                                     ignore: Sequence[Sequence[int]] = None) -> tuple[int]:
+        """
+        Creates a single new flow id for a given hop, based
```

Comment:
Should we do the PEP recommendation of having a one-line summary and a more complete description in another paragraph?

Reply:
Agreed, I changed it, let me know if you have a more complete description.